src/downshift.js

```diff
@@ -983,6 +990,10 @@ class Downshift extends Component {
   }
 
   componentDidUpdate(prevProps, prevState) {
```

Comment:
Should we add this to a `componentDidMount` as well?

Reply:
Yep that would make sense!